src/text/test/contains-cases.ts

```diff
@@ -0,0 +1,9 @@
+// charAt test cases
```

Comment:
Comment should read "contains test cases".

Reply:
fixed